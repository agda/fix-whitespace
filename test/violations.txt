Trailing space 

Trailing tab    

Missing newline at end of file