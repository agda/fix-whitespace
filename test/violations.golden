Violations:
test/violations.txt:1: Trailing·space·
test/violations.txt:3: Trailing·tab····
